homeassistant/components/nmbs/sensor.py

```diff
@@ -205,9 +205,13 @@ def extra_state_attributes(self):
 
         delay = get_delay_in_minutes(self._attrs["departure"]["delay"])
         departure = get_time_until(self._attrs["departure"]["time"])
+        canceled = int(self._attrs["departure"]["canceled"])
+
+        if canceled != 0:
+            departure = "Canceled"
 
         attrs = {
-            "departure": f"In {departure} minutes",
+            "departure": f"In {departure} minutes" if canceled != 0 else departure,
             "departure_minutes": departure,
```

Comment:
This won't have minutes if the departure is canceled. We shouldn't set this attribute if the departure is canceled.